### DISCUSSION TITLE
Missing fedora-release package

### DISCUSSION THREAD

**Discussion by gtirloni** [WhitewaterFoundry/Fedora-Remix-for-WSL#159] [General]
This is more of a question than a bug report, sorry if this is the wrong place.

I was trying to install the Node.js packages from NodeSource from [here](https://github.com/nodesource/distributions).

Their install script fails because it looks for the `fedora-release` package that usually exists in Fedora. It seems like a reasonable assumption, I think.

I'm thinking the missing `fedora-release` package is deliberate in WSL but I'm not sure. Is it something expected?

**Comment by crramirez** [comment]
Hello @gtirloni,

A better place for it is in the discussion tab.

Yes, `fedora-release` is not installed on purpose, not for technical but for legal reasons. This is to comply with the Fedora Remix terms. The package `generic-release` is installed instead. You can if you wish install the fedora-release package with:

`sudo dnf -y install fedora-release --allowerasing`

But I personally recommend you to install Node.js with a version manager like **nvm** or **n**:

`curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash`

Restart Fedora Remix and then install node with:

`nvm install node`

Hope it helps

**Reply by gtirloni** [reply]
Thank you, much appreciated.
